vscode/src/completions/create-inline-completion-item-provider.ts

```diff
@@ -111,3 +114,35 @@ export async function createInlineCompletionItemProvider({
         },
     }
 }
+
+export async function getInlineCompletionItemProviderFilters(
+    autocompleteLanguages: Record<string, boolean>
+): Promise<vscode.DocumentFilter[]> {
+    const { '*': isEnabledForAll, ...perLanguageConfig } = autocompleteLanguages
+
+    // Enable for every known langauge ID if it's not explicitly disabled in `perLanguageConfig`.
+    if (isEnabledForAll) {
+        const languageIds = await vscode.languages.getLanguages()
+
+        return languageIds
+            .map(language => {
+                if (perLanguageConfig[language] === undefined || perLanguageConfig[language] === true) {
+                    return { language, scheme: 'file' }
+                }
+
+                return null
+            })
+            .filter(isDefined)
```

Comment:
You can do this more simply with `flatMap`
```suggestion
            .flatMap(language => perLanguageConfig[language] === undefined || perLanguageConfig[language] === true ? [{ language, scheme: 'file' }] : [])
```